gapis/api/vulkan/find_issues.go

```diff
@@ -290,16 +290,12 @@ func (t *findIssues) Flush(ctx context.Context, out transform.Writer) error {
 			if issue.Command == api.CmdNoID {
 				// The debug report is issued for state rebuilding command
 				// TODO: Fix all the errors reported for initial commands.
-				// TODO: Provide a way for the UI to distinguish these issues
-				// from issues on command 0.
-				issue.Command = api.CmdID(0)
-				issue.Error = fmt.Errorf("[State rebuilding command : %s]	", msg)
+				log.E(ctx, "Error in state rebuilding command : %s", msg)
```

Comment:
We should probably log with the same severity as the validation layers?

Reply:
I am not sure about this one. As these are our issues instead of the applications, maybe it might be better to consider them as errors regardless of severity. We may still print the severity in the log message though.

Reply:
I have decided to put it behind a config flag to not confuse our users.